diff --git a/hack/dind-cluster.sh b/hack/dind-cluster.sh
index 71381efb5c..35c8414720 100755
--- a/hack/dind-cluster.sh
+++ b/hack/dind-cluster.sh
@@ -415,7 +415,7 @@ function start-container() {
   volumes=""
 
   ${DOCKER_CMD} run -dt ${volumes} -v "${config_root}:${deployed_config_root}" --privileged \
-                --name="${name}" --hostname="${name}" "${image}" > /dev/null
+                --name="${name}" --dns-opt timeout:1 --dns-opt ndots:5 --dns 172.30.0.1 --dns 8.8.8.8 --dns-search default.svc.cluster.local --dns-search svc.cluster.local --dns-search cluster.local -v /sys/fs/cgroup:/sys/fs/cgroup:rw --hostname="${name}" "${image}" > /dev/null
 }
 
 function delete-node () {
@@ -515,7 +515,7 @@ function pause() {
   check-containers "pause"
 
   for cid in $( ${DOCKER_CMD} ps -qa --filter "name=${MASTER_NAME}|${NODE_PREFIX}" ); do
-    ${DOCKER_CMD} stop "${cid}" > /dev/null &
+    ${DOCKER_CMD} stop -t 120 "${cid}" > /dev/null &
   done
 
   wait
