diff --git a/hack/build-dind-images.sh b/hack/build-dind-images.sh
index dcd1a9aca4..86b462317e 100755
--- a/hack/build-dind-images.sh
+++ b/hack/build-dind-images.sh
@@ -14,6 +14,6 @@ function cleanup() {
 }
 trap "cleanup" EXIT
 
-os::build::image openshift/dind        "${OS_ROOT}/images/dind"
-os::build::image openshift/dind-node   "${OS_ROOT}/images/dind/node"
-os::build::image openshift/dind-master "${OS_ROOT}/images/dind/master"
+os::build::image gustavonalle/openshift-dind        "${OS_ROOT}/images/dind"
+os::build::image gustavonalle/openshift-dind-node   "${OS_ROOT}/images/dind/node"
+os::build::image gustavonalle/openshift-dind-master "${OS_ROOT}/images/dind/master"
diff --git a/hack/dind-cluster.sh b/hack/dind-cluster.sh
index 71381efb5c..a840fb61eb 100755
--- a/hack/dind-cluster.sh
+++ b/hack/dind-cluster.sh
@@ -99,7 +99,7 @@ function start() {
   # if docker is running remotely.  The openshift/dind-node image was chosen
   # due to its having sysctl installed.
   ${DOCKER_CMD} run --privileged --net=host --rm -v /lib/modules:/lib/modules \
-                openshift/dind-node bash -e -c \
+                gustavonalle/openshift-dind-node bash -e -c \
                 '/usr/sbin/modprobe openvswitch;
                 /usr/sbin/modprobe overlay 2> /dev/null || true;'
 
@@ -846,9 +846,9 @@ ADDITIONAL_BRIDGE_NAME="${ADDITIONAL_BRIDGE_NAME:-os-addbr-1}"
 ADDITIONAL_NETWORK_NUM=18
 ADDITIONAL_IFACE_NAME="${ADDITIONAL_IFACE_NAME:-eth1}"
 
-BASE_IMAGE="openshift/dind"
-NODE_IMAGE="openshift/dind-node"
-MASTER_IMAGE="openshift/dind-master"
+BASE_IMAGE="gustavonalle/openshift-dind"
+NODE_IMAGE="gustavonalle/openshift-dind-node"
+MASTER_IMAGE="gustavonalle/openshift-dind-master"
 ADDITIONAL_ARGS=""
 
 OVN_ROOT="${OVN_ROOT:-}"
diff --git a/images/dind/Dockerfile b/images/dind/Dockerfile
index e3568cff4d..f4181af1ca 100644
--- a/images/dind/Dockerfile
+++ b/images/dind/Dockerfile
@@ -81,4 +81,6 @@ VOLUME ["/var/lib/docker"]
 # to ensure cgroups are mounted with read-write permissions.
 RUN ln /usr/sbin/init /usr/sbin/dind_init
 
+RUN chown 1001.1001 /etc/docker/certs.d
+
 CMD ["/usr/sbin/dind_init"]
diff --git a/images/dind/master/Dockerfile b/images/dind/master/Dockerfile
index aa0653ab30..2a45094143 100644
--- a/images/dind/master/Dockerfile
+++ b/images/dind/master/Dockerfile
@@ -4,7 +4,7 @@
 # The standard name for this image is openshift/dind-master
 #
 
-FROM openshift/dind-node
+FROM gustavonalle/openshift-dind-node
 
 RUN ln -sf /data/openshift /usr/local/bin/ && \
     ln -sf /data/hypershift /usr/local/bin/
diff --git a/images/dind/node/Dockerfile b/images/dind/node/Dockerfile
index 5cef27e46c..c8dcbab490 100644
--- a/images/dind/node/Dockerfile
+++ b/images/dind/node/Dockerfile
@@ -4,7 +4,7 @@
 # The standard name for this image is openshift/dind-node
 #
 
-FROM openshift/dind
+FROM gustavonalle/openshift-dind
 
 ## Install packages
 RUN dnf -y update && dnf -y install\
@@ -99,6 +99,8 @@ COPY crio-node.sh /usr/local/bin/
 COPY crio-node.service /etc/systemd/system/
 RUN systemctl enable crio-node.service
 
+RUN chown -R 1001:1001 /etc/docker/certs.d/
+
 # Default crio storage to vfs. overlay on overlayfs is not supported in crio.
 RUN echo "CRIO_STORAGE_OPTIONS=--storage-driver vfs" > /etc/sysconfig/crio-storage
 RUN sed -i 's/^plugin_dir .*/plugin_dir = "\/opt\/cni\/bin\/"/g' /etc/crio/crio.conf
